src/os/bluestore/BlueStore.cc

```diff
@@ -5380,6 +5383,14 @@ int BlueStore::_balance_bluefs_freespace(PExtentVector *extents)
   bluefs->get_usage(&bluefs_usage);
   ceph_assert(bluefs_usage.size() > bluefs_shared_bdev);
 
+  _clear_alert(ALERT_BLUEFS_SPILLOVER);
+  if (bluefs_shared_bdev == BlueFS::BDEV_SLOW) {
+    auto& p = bluefs_usage[bluefs_shared_bdev];
+    if (p.first != p.second) {
+      _set_alert(ALERT_BLUEFS_SPILLOVER);
```

Comment:
I'm concerned about this one.  It is totally valid to have an OSD with a big HDD and a small about of SSD space, with maybe only L0 and L1 on it.  It's faster than having no SSD at all.   Spillover isn't a "health" issue, it's a symptom of the cost/performance tradeoff the admin made.  It seems like we should make spillover visible in some other way than a health warning...

Reply:
@markhpc @jdurgin @neha-ojha @LenzGr

Reply:
@liewegas - may be make "spillover" warning configurable? Simple enable/disable switch or spillover size threshold parameter to start warning at?